include/osquery/tables.h

```diff
@@ -177,7 +179,7 @@ struct Constraint {
  * users table. This options structure will allow the table implementations
  * to communicate these subtleties to the user.
  */
-enum ColumnOptions {
+enum class ColumnOptions {
```

Comment:
This is a change, adding `class` here allows for `DEFAULT` to be isolated within this enum namespace.